### PR TITLE
feat: expose jdbc connector support

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: pkg/entity/data_connector/cmd.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ const (
 
 var longDoc = util.LongDocsUsage(`
 A Data Connector represents a location from which data can be read, or to which data can be written.  For
-example, an AWS S3 bucket or a Google Cloud Storage bucket. By itself, a Data Connector does nothing.  A Data Connector
+example, an AWS S3 bucket or a Google Cloud Storage bucket or a JDBC database. By itself, a Data Connector does nothing.  A Data Connector
```
Comment (ivan-p92) [inline pkg/entity/data_connector/cmd.go:17]:
```suggestion
example, an AWS S3 bucket, a Google Cloud Storage bucket or a JDBC database connection. By itself, a Data Connector does nothing.  A Data Connector
```

File: pkg/entity/data_connector/jdbc_connection.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/strmprivacy/api-definitions-go/v2/api/entities/v1"
+	"os"
+	"strings"
+	"strmprivacy/strm/pkg/common"
+)
+
+const (
+	longDocJdbc = `Creates a data connector for a jdbc database.
```
Comment (ivan-p92) [inline pkg/entity/data_connector/jdbc_connection.go:14]:
```suggestion
	longDocJdbc = `Creates a data connector for a JDBC-compatible database.
```

File: pkg/entity/data_connector/jdbc_connection.go [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+			entities.DatabaseType_MYSQL.String(),
+			entities.DatabaseType_POSTGRES.String(),
+			entities.DatabaseType_BIGQUERY.String(),
+		)))
+	}
+
+	return databaseType
+}
+
+func loadPrivateKey(jdbcUrl string, databaseType entities.DatabaseType) string {
```
Comment (ivan-p92) [inline pkg/entity/data_connector/jdbc_connection.go:72]: Will this private key file be present on the CLI user's machine?

**Review by ivan-p92 — APPROVED** [review]

File: pkg/entity/data_connector/jdbc_connection.go (hunk 3 above)
Comment (bobvandenhoogen) [reply pkg/entity/data_connector/jdbc_connection.go:72]: yes it should

**Pull request merged**